packages/client/README.md

```diff
@@ -88,6 +88,33 @@ Setting a new admin fee:
 // Any number between 0 and 1, inclusive
 const receipt = await dataUnion.setAdminFee(0.4)
 ```
+
+If the Data Union is set up to use the [default join server](../default-join-server/README.md) then members can join the Data Union by giving a correct secret.
+
+Admin can add secrets that allow anyone to join, as well as revoke those secrets, using the following functions:
+```js
+await dataUnion.createSecret() // returns the newly created secret
+await dataUnion.createSecret('user XYZ') // admin can also give the secret a more human-readable name
+await dataUnion.deleteSecret(secretId) // id is what was returned from createSecret
```

Comment:
@jtakalai: `createSecret` does not return any id property. We should have one for `deleteSecret`.

Reply:
it's called `secret`, that property is the "id". Maybe it should be called id... but then it's also the actual thing that is passed. A bit confusing maybe.